### PR TITLE
PublicDashboards: Fix panel render

### PR DESCRIPTION
**Pull request by juanicabanas**
**What is this feature?**

After [this PR](https://github.com/grafana/grafana/pull/91888), panels are not well rendered in public dashboard.
New CSS is added in `PublicDashboardScenePage`.

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

I think we should use the same CSS as DashboardSceneRenderer, somehow, instead of repeating it.

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Thanks! forgot to test this page :(

File: public/app/features/dashboard-scene/pages/PublicDashboardScenePage.tsx [hunk 1]
```diff
@@ -99,16 +100,42 @@ function PublicDashboardSceneRenderer({ model }: SceneComponentProps<DashboardSc
...
-        <bodyToRender.Component model={bodyToRender} />
-      </div>
+      <NativeScrollbar divId="page-scrollbar" autoHeightMin={'100%'}>
+        <div className={styles.pageContainer}>
+          <div className={styles.canvasContent}>
+            <bodyToRender.Component model={bodyToRender} />
+          </div>
+        </div>
+      </NativeScrollbar>
       <PublicDashboardFooter />
```
Comment (torkelo) [inline public/app/features/dashboard-scene/pages/PublicDashboardScenePage.tsx:110]: I am confused why this needed? I am testing public dashboards no in latest main and see no issues

Comment (torkelo) [reply public/app/features/dashboard-scene/pages/PublicDashboardScenePage.tsx:110]: think a simple "    display: flex;"  on the body class fixes it, ash will open a new PR

**Comment by juanicabanas** [comment]
Already fixed [here](https://github.com/grafana/grafana/pull/92187)

**Pull request closed without merging**